special-interest-groups/sig-k8s/member-list.md

```diff
@@ -0,0 +1,15 @@
+# SIG K8S Member List
+
+**NOTE**:
+
+* This member list is updated on a bi-weekly basis.
+* Last update time: 2020-02-21
+
+## Tech Leads
+
+Leads:
+* [tennix](https://github.com/tennix), [PingCAP](https://pingcap.com/en/)
```

Comment:
I suggest @cofyc and @Yisaer as the lead and co-lead.

Reply:
updated. PTAL